lib/k8sClient.js

```diff
@@ -2,6 +2,8 @@
 
 const k8s = require('k8s'),
     debug = require('debug')('helmit:k8sClient');
+var STRICT_SSL = process.env.STRICT_SSL || 'doc,iso';
+var AUTH_TOKEN = process.env.AUTH_TOKEN || '';
```

Comment:
we are using the same token for all clusters? Shouldn't this be namespaces for each barge?